Custom Device Source/Scripting Examples/System Definition Files/Base with Scan Engine Chassis.nivssdf

```diff
@@ -0,0 +1,2553 @@
+														<Double>1</Double>
+													</Elem>
+													<Elem Key="RiskOfDamage">
+														<Double>2</Double>
+													</Elem>
+												</Dictionary>
+											</Property>
+										</Properties>
+										<Errors />
+										<DefaultValue>
+											<Elem>-1</Elem>
+										</DefaultValue>
+									</Channel>
+								</Section>
+								<Section Name="FanVoltageSensor" TypeGUID="96faf7a1-7cab-4db2-971e-68b73536c883">
+									<Description />
+									<Properties />
+									<Errors />
+									<Channel Name="SensorReading" TypeGUID="92ef77b2-7367-42f8-a914-3eb37c710e2e" RowDim="1" ColDim="1" Units="Double" BitFields="1">
+										<Description>The current voltage of the fan in unit voltage.</Description>
+										<Properties />
+										<Errors />
+										<DefaultValue>
+											<Elem>0</Elem>
+										</DefaultValue>
+									</Channel>
+									<Channel Name="SensorLowerCritical" TypeGUID="92ef77b2-7367-42f8-a914-3eb37c710e2e" RowDim="1" ColDim="1" Units="Double" BitFields="1">
+										<Description>The fan's lower threshold of the operating voltage in unit voltage.</Description>
+										<Properties />
+										<Errors />
+										<DefaultValue>
+											<Elem>0.08</Elem>
+										</DefaultValue>
+									</Channel>
+									<Channel Name="SensorUpperCritical" TypeGUID="92ef77b2-7367-42f8-a914-3eb37c710e2e" RowDim="1" ColDim="1" Units="Double" BitFields="1">
+										<Description>The fan's upper threshold of the operating voltage in unit voltage.</Description>
+										<Properties />
+										<Errors />
+										<DefaultValue>
+											<Elem>0.12</Elem>
+										</DefaultValue>
+									</Channel>
+									<Channel Name="HealthState" TypeGUID="92ef77b2-7367-42f8-a914-3eb37c710e2e" RowDim="1" ColDim="1" Units="Enum" BitFields="1">
+										<Description>Health State of the chassis fan.
+
+-1: Unknown
+0: Normal
+1: OutOfSpec
+2: RiskOfDamage</Description>
+										<Properties>
+											<Property Name="Value Table">
+												<Dictionary>
+													<Elem Key="Unknown">
+														<Double>-1</Double>
+													</Elem>
+													<Elem Key="Normal">
+														<Double>0</Double>
+													</Elem>
+													<Elem Key="OutOfSpec">
+														<Double>1</Double>
+													</Elem>
+													<Elem Key="RiskOfDamage">
+														<Double>2</Double>
+													</Elem>
+												</Dictionary>
+											</Property>
+										</Properties>
+										<Errors />
+										<DefaultValue>
+											<Elem>-1</Elem>
+										</DefaultValue>
+									</Channel>
+								</Section>
+							</Section>
+						</Section>
+					</Section>
+				</Section>
+				<Section Name="Custom Devices" TypeGUID="03D3BB79-1485-13A6-5605EB7AFD7405AC">
+					<Description />
+					<Properties />
+					<Errors />
+					<Section Name="Scan Engine and EtherCAT" TypeGUID="A04A67D4-9B64-052B-A031-6B05377B9B10">
```

Comment:
Is there a reason to have a system definition with the Scan Engine custom device pre-added?

We needed a separate system definition for R+F, because two custom devices were being scripted and each example only touched one custom device. I don't think you'll need this one.

Reply:
This is a place holder for a later example that will focus on reading existing modules rather than creating a new one. It could be added later.

Reply:
I'm fine with adding it now, but we need to remove the other leftover items (SLSC Switch CD) from this system definition as well.

Reply:
i removed it for now